### PR TITLE
Fix example03 - new bidding_strategies structure

### PR DESCRIPTION
**Pull request by maurerle**
* change scenario files structure according to proposal in #95

### CODE REVIEW

File: assume/common/scenario_loader.py [hunk 1]
```diff
@@ -378,13 +375,9 @@ async def load_scenario_folder_async(
     # if vre generation is provided, add them to the vre units
     # if we have RL strategy, add price forecast to unit_params
     def powerplant_callback(unit_name, unit_params):
-        # check if we have RL bidding strategy
-        if (
-            unit_params["bidding_strategies"]["energy"] == "rl_strategy"
-            and world.forecast_providers["EOM"].price_forecast_df is not None
-        ):
-            unit_params["price_forecast"] = world.price_forecast["mcp"]
-            unit_params["res_demand_forecast"] = world.price_forecast["residual_demand"]
```
Comment (maurerle) [inline assume/common/scenario_loader.py:387]: `unit_params["res_demand_forecast"] = world.price_forecast["residual_demand"]` went missing here? unneeded?

File: assume/common/scenario_loader.py [hunk 2]
```diff
@@ -242,7 +236,11 @@ async def load_scenario_folder_async(
         file_name="demand_units",
     )
 
-    heatpump_units = load_file(path=path, config=config, file_name="heatpumps")
+    heatpump_units = load_file(
+        path=path,
+        config=config,
+        file_name="heatpumps",
```
Comment (maurerle) [inline assume/common/scenario_loader.py:242]:
```suggestion
        file_name="heatpump_units",
```

The file_name is now `heatpump_units` which is consistent with everything else

File: assume/common/scenario_loader.py (hunk 1 above)
Comment (nick-harder) [reply assume/common/scenario_loader.py:387]: this was needed for the learning agents. We will make a clean implementation with Kim once we Strat with it next week

**Comment by nick-harder** [comment]
@kim-mskw is now good to go :-)

**Review by nick-harder — APPROVED** [review]

**Pull request merged**